### DISCUSSION TITLE
external_id Search Not Working

### DISCUSSION THREAD

**Discussion by richwalm** [firefly-iii/firefly-iii#4780] [Firefly III]
I have a script to import transactions from my bank which uses a UUID as a unique ID. I add this to Firefly using the `external_id` field.

However, I'm unable to search for them using a `external_id:123e4567-e89b-12d3-a456-426614174000` like search query. Both firefly-iii's API & web interface search comes back with no results. I've dived into the `journal_meta` table and can confirm that the UUIDs are visible there.

Is this search query option working for anyone else? I'm running 5.5.11.

**Comment by JC5** [comment]
You could try `external_id:"xxxx"` but I'll check the code to be sure.

**Reply by richwalm** [reply]
Thanks. Had tried with quotes but that's not working either for me.

**Comment by JC5** [comment]
Alright, I'll do some digging. I noticed something similar in another search query.

**Reply by richwalm** [reply]
Hey JC5, did you find anything? I've attempted to do some source diving myself but I'm not much of a PHP person.

**Reply by JC5** [reply]
Not yet, stay tuned!

**Comment by JC5** [comment]
Will be fixed in the next release 👍🏻
